app/src/main/res/layout/recent_tabs_header.xml

```diff
@@ -6,29 +6,35 @@
     xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
There's something weird on this header compare to the others. Also, all the "Show all" button seems to have a weird constraint.

<img width="441" alt="Screen Shot 2021-09-22 at 6 12 50 PM" src="https://user-images.githubusercontent.com/1190888/134429061-eadfbf86-3762-405a-9631-7e50d47c43d7.png">

Reply:
Another side effect of the current design is that the click gesture on the "Show all" shows that hard right edge compare to having it centered. Might be worthwhile to show it to @topotropic

Reply:
This can be dealt with as a followup.